providers/mailgun/src/lib/mailgun.provider.ts

```diff
@@ -44,6 +44,8 @@ export class MailgunEmailProvider implements IEmailProvider {
       to: emailOptions.to,
       subject: emailOptions.subject,
       html: emailOptions.html,
+      cc: emailOptions.cc ? emailOptions.cc[0] : undefined,
+      bcc: emailOptions.bcc ? emailOptions.bcc[0] : undefined,
```

Comment:
Mailgun doesn't accept `cc` and `bcc` as a list?

Reply:
Not what I could find...

Reply:
Then I wouldn't apply BCC and CC to Mailgun and the other provider. Or if doing it inform the user that only one email is allowed per field. We will get a lot of questions from people asking why the mail was not delivered to all BCC and CC. 😭

Reply:
@scopsy should we mention this in the docs maybe?
ping @iampearceman and @jainpawan21

Reply:
Never mind found a way of doing it interfaces was not clear enough